src/current/cockroachcloud/managed-backups.md

```diff
@@ -34,6 +34,10 @@ Refer to the [Cloud Console](#restore-a-basic-cluster) section for details on vi
 
 ### Standard and Advanced clusters
 
+{{site.data.alerts.callout_info}}
```

Comment:
I wonder if this is necessary, given it is supported in all "supported" versions of CRDB https://www.cockroachlabs.com/docs/releases/release-support-policy#supported-versions?

Reply:
could we just call out that its in all supported versions?